AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5158,6 +5158,7 @@ arctic.de#$#.acris-cookie-consent ~ .modal-backdrop { display: none !important;
 maisonapart.com#$?#html[-ext-has="body"] { margin-top: auto!important; }
 !
 kjell.com#?##content-container > div[class] > div[class]:matches-css(z-index: 10) > div[class] > div[class] > h4:contains(cookies):upward(4)
+glosbe.com#?#body[class*="cookie-consent"] > .fixed:contains(cookies)
```

Comment:
It doesn't work on my end. Try `glosbe.com#?#body > div[class] > div[id="cookieconsent-description"]:upward(1)`.

Reply:
-_- Check properly. It works fine on my end.
<details>

![image](https://user-images.githubusercontent.com/76908410/110427919-6d066800-80e3-11eb-8fbe-8b252524bca5.png)
</details>

Reply:
I removed the banner leftovers too.

Reply:
Your rule doesn't do anything on my end.
<details>
![image](https://user-images.githubusercontent.com/76908410/110428240-ed2ccd80-80e3-11eb-8c08-77f77fa677e9.png)
</details>

Reply:
> It works fine on my end

According to the code, it can't be be applied. I've already checked it, that's why I requested changes.

<details><summary>Your rule:</summary>

![image](https://user-images.githubusercontent.com/33488229/110428835-0705ea80-80bb-11eb-9c6c-d8a87deb8524.png)

</details><br/>

<details><summary>My variant:</summary>

![image](https://user-images.githubusercontent.com/33488229/110429027-5ea45600-80bb-11eb-8a3e-e62d4b0ad711.png)

</details><br/>

Reply:
![image](https://user-images.githubusercontent.com/76908410/110429415-e30bce80-80e5-11eb-9f9e-6357dd137954.png)

Reply:
Btw, what browser is that? Chrome? It looks very weird, with the star button on the left.

Reply:
Show the code of this element with opened devtools.

Reply:
@zloyden Trust me it works. I won't add redundant rules.
<details>

![image](https://user-images.githubusercontent.com/76908410/110430550-81e4fa80-80e7-11eb-8441-2f089ff235c8.png)
<details>

Reply:
It's Cent browser. Btw, your rule works only with `EN` interface, but switch it to the `Arabic`, for instance.

>I won't add redundant rules.

You didn't show this properly. It should be opened.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/110430953-2d795500-80be-11eb-8633-423f48f334a1.png)
</details><br/>

Reply:
@zloyden The problem is that I do not know to use the upward selector as the AdGuard Knwoledge base remains unclear. I made the rule by what I know how to use.
<details>

![image](https://user-images.githubusercontent.com/76908410/110431534-e48ac600-80e8-11eb-92cf-124dba37efab.png)
<details>

Reply:
@zloyden In my native language (Polish).

<details>

![image](https://user-images.githubusercontent.com/76908410/110432559-413ab080-80ea-11eb-9944-2dfa6a384974.png)
</details>

Reply:
@zloyden I will change to your rule. I could reproduce it with Russian.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/77012#discussion_r589999719 I have the same, so there is no difference and my rule should work on your end. I've checked this with extension/app and it blocks the cookies. I don't know why it doesn't work on your end.
<details><summary>Screenshot with app and Arabian interface:</summary>

![image](https://user-images.githubusercontent.com/33488229/110432904-d628b400-80c0-11eb-9ca5-7fb41b12bb0b.png)

</details><br/>

Anyway, your variant with `:contains` only works with `English text` that match `cookies`.
If you don't exchange the rule, I'll change it directly.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/77012#discussion_r590011139

Reply:
> I will change to your rule. I could reproduce it with Russian.

Ok.